opentelemetry/proto/profiles/v1experimental/pprofextended.proto

```diff
@@ -307,12 +296,6 @@ message Mapping {
   // disk for the main binary and shared libraries, or virtual
   // abstractions like "[vdso]".
   int64 filename = 5;  // Index into string table
-  // A string that uniquely identifies a particular program version
-  // with high probability. E.g., for binaries generated by GNU tools,
-  // it could be the contents of the .note.gnu.build-id field.
-  int64 build_id = 6;  // Index into string table
-  // Specifies the kind of build id. See BuildIdKind enum for more details [optional]
-  BuildIdKind build_id_kind = 11;
```

Comment:
Similar to https://github.com/open-telemetry/opentelemetry-proto/pull/578#discussion_r1736133541, indexing of fields in messages will be updated once before the profiling signal becomes stable.